v5.15.1 (#5699)
mui/mui-x: CHANGELOG.md

All notable changes to this project will be documented in this file.
See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.

## 5.15.1

_Aug 4, 2022_

We'd like to offer a big thanks to the 9 contributors who made this release possible. Here are some highlights ✨:

- 📚 New [page presenting the `apiRef`](https://mui.com/x/react-data-grid/api-object/) (#5273) @flaviendelangle

> - ✨ Improve start edit UX (#5511) @oliviertassinari

Reviewer: Only to avoid repeating "improve".

```suggestion
- ✨ Better keyboard support for start editing cells (#5511) @oliviertassinari
```

- 🌍 Improvements to different locales
- 🐞 Bugfixes

### `@mui/x-data-grid@v5.15.1` / `@mui/x-data-grid-pro@v5.15.1` / `@mui/x-data-grid-premium@v5.15.1`

#### Changes

- [DataGrid] Improve start edit UX (#5511) @oliviertassinari

> - [DataGrid] Add `initialOpen` prop to `GridEditSingleSelectCell` to allow overriding default behavior on cell edit mode (#5645) @shapaaa

Reviewer 2: Not sure I get the point of this PR
```suggestion
- [DataGrid] Add `initialOpen` prop to `GridEditSingleSelectCell` to chose which menu open in row edit mode (#5645) @shapaaa
```

Author: It could actually work in both cell and row edit modes, so maybe

```suggestion
- [DataGrid] Add `initialOpen` prop to `GridEditSingleSelectCell` to allow overriding initial open state (#5645) @shapaaa
```

- [DataGrid] Forward `ref` to root element in `GridEditInputCell` (#5631) @Zenoo

> - [DataGrid] Toggle open state when clicking on buttons in `GridToolbar` (#5503) @cherniavskii

Reviewer: ```suggestion
- [DataGrid] Toggle open state when clicking on buttons in the `GridToolbar` (#5503) @cherniavskii
```

- [DataGrid] Improve German (de-DE) locale (#5586) @sebastianfrey
- [DataGrid] Improve Korean (ko-KR) locale (#5668) @Einere
- [DataGrid] Complete Italian (it-IT) locale (#5487) @mamodev

> 

Reviewer 2: ```suggestion

### `@mui/x-date-pickers@v5.0.0-beta.3` / `@mui/x-date-picker-pro@5.0.0-beta.3`

#### Changes

- [DatePicker] Customize day formatter in the calendar (#5373) @alexfauquette

```
